docs/integrations/bitbucketServer/discovery.md

```diff
@@ -93,27 +84,3 @@ catalog:
     The amount of time that should pass before the first invocation happens.
   - **`scope`** _(optional)_:
     `'global'` or `'local'`. Sets the scope of concurrency control.
-
-## Custom location processing
```

Comment:
Please log a Feature request to add this as an extension point to the provider. I'm not sure how much this is used but we should have something to track this.

Reply:
OK I will create a feature request.

Reply:
Submitted: #25696